extensions/binding/org.eclipse.smarthome.binding.weatherunderground/OSGI-INF/WeatherUndergroundDiscoveryService.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Delay changing this to annotations or delay adding the info to the documentation?